docs/user-guide/kubernetes.md

```diff
@@ -330,6 +330,59 @@ echo "$(minikube ip) traefik-ui.minikube" | sudo tee -a /etc/hosts
 
 We should now be able to visit [traefik-ui.minikube](http://traefik-ui.minikube) in the browser and view the Træfik Web UI.
 
+## Basic Authentication
+Is possible to add additional authentication annotations in the Ingress rule. The source of the authentication is a secret that contains usernames and passwords inside the the key auth.
+
+* Basic authentication only
+* Realm not configurable; only traefikdefault
+* Secret is in same namespace as ingress rule
+* Secret must contain only single file
```

Comment:
The whole paragraph basically duplicates what can already be found in the [Kubernetes configuration documentation](https://github.com/containous/traefik/blob/07c6e335980a80cc58dcbed686809e188b16dcf1/docs/configuration/backends/kubernetes.md#authentication). Instead of repeating, I suggest we generally mention that a certain number of restrictions exist and leave a pointer where to find them.